istack-commons/pom.xml

```diff
@@ -80,41 +86,210 @@
         <module>soimp</module>
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
+        <dependency>
+            <groupId>jakarta.activation</groupId>
+            <artifactId>jakarta.activation-api</artifactId>
+            <version>1.2.1</version>
+        </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.12</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant</artifactId>
+                <version>1.10.2</version>
+                <scope>compile</scope>
```

Comment:
compile is default, so either remove or use everywhere...

Reply:
Removed.